### PR TITLE
Add ViromeQC

### PR DESCRIPTION
**Pull request by CarsonJM**
Adding ViromeQC. A package that determines viral enrichment in metagenomic samples (https://github.com/SegataLab/viromeqc)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by CarsonJM** [comment]
@BiocondaBot please add label

File: recipes/viromeqc/meta.yaml
Comment (bgruening) [inline recipes/viromeqc/meta.yaml:18]: I don't think you need git, do you?

File: recipes/viromeqc/meta.yaml
Comment (bgruening) [inline recipes/viromeqc/meta.yaml:21]: Please remove the strict pinning, they should not be needed.

File: recipes/viromeqc/meta.yaml
Comment (CarsonJM) [reply recipes/viromeqc/meta.yaml]: @bgruening You are correct, git has been removed in the most recent commit.

File: recipes/viromeqc/meta.yaml
Comment (CarsonJM) [reply recipes/viromeqc/meta.yaml]: @bgruening Strict pinning has been removed. I initially thought there were some version restrictions but found that wasn't the case after some testing this morning.

**Comment by CarsonJM** [comment]
@bgruening Are there any other changes you would like me to make before merging? Also, thank you for your previous feedback!

**Comment by CarsonJM** [comment]
@bioconda/core Would it be possible to get another review for this PR? Thank you!

File: recipes/viromeqc/build.sh
Comment (bgruening) [inline recipes/viromeqc/build.sh:9]: the echos can be removed

Comment (CarsonJM) [reply recipes/viromeqc/build.sh]: Echos have been removed!

**Review by dpryan79 — APPROVED** [review]

**Comment by CarsonJM** [comment]
Thank you @bgruening and @dpryan79 for your help!

**Pull request merged**
